README.md

```diff
@@ -117,84 +146,97 @@ func getServicePrincipalToken() (adal.OAuthTokenProvider, error) {
+-mindepth 3` to list all available service packages.
 
-**SDK versions** are set with repository-wide [tags][]. We try to adhere to
-[semver][], so although we've long moved past version zero, we continue to
-add `-beta` to versions because **this package is still in preview.**
+[services_godoc]:       https://godoc.org/github.com/Azure/azure-sdk-for-go/services
 
-[tags]: https://github.com/Azure/azure-sdk-for-go/tags
-[semver]: https://semver.org
+### Profiles
 
-Azure **API versions** are typically a date string of form
-`yyyy-mm-dd[-preview|-beta]`. Whatever SDK version you use, you must also
-specify an API version in your import path, for example:
+Azure **API profiles** specify subsets of Azure APIs and versions. Profiles can provide:
 
-```go
-import "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2016-03-01/compute"
-import "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network"
-```
+* **stability** for your application by locking to specific API versions; and/or
+* **compatibility** for your application with Azure Stack and regional Azure datacenters.
 
-For a list of all available services and versions see [godoc][services_godoc]
-or start with `find ./services -type d -mindepth 3` within this repo. 
-
-Azure **API profiles** specify a subset of APIs and versions offered in
-specific Azure regions and Azure Stack. The 2017-03-09 profile is intended for
-use with Azure Stack and includes compatible Compute, Network, Storage and
-Group management APIs. You can use it as follows:
+In the Go SDK, profiles are available under the `profiles/` path and their
+component API versions are aliases to the true service package under
+`services/`. You can use them as follows:
 
 ```go
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/compute/mgmt/compute"
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/network/mgmt/network"
-import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/resources/mgmt/..."
 import "github.com/Azure/azure-sdk-for-go/profiles/2017-03-09/storage/mgmt/storage"
 ```
 
-We also provide two special profiles: `latest` and `preview`. These will always
-refer to the most recent stable or preview API versions for each service. For
-example, to automatically use the most recent Compute APIs, use one of the following
-imports:
+The 2017-03-09 profile is the only one currently available and is for use with Azure
+Stack. More profiles are under development.
+
+In addition to versioned profiles, we also provide two special profiles
+`latest` and `preview`. These *always* include the most recent respective stable or
+preview API versions for each service, even when updating them to do so causes
+breaking changes.
+
+The `latest` and `preview` profiles can help you stay up to date with API
+updates as you build applications. Since they are by definition not stable,
+however, they **should not** be used in production apps. Instead, choose the
```

Comment:
To be super explicit, I may add particular mention here that the SemVer on the repository doesn't cover these profiles.

Reply:
okay will add